src/components/ProjectIdeaForm.tsx

```diff
@@ -39,6 +40,10 @@ const ProjectIdeaForm: React.FC<ProjectIdeaFormProps> = ({
 	const [projectDescription, setProjectDescription] = useState("");
 	const [feedbackQuestion, setFeedbackQuestion] = useState("");
 	const [selectedTags, setSelectedTags] = useState<string[]>([]);
+	// const [generalQuestion, setGeneralQuestion] = useState("");
+	const [uploads, setUploads] = useState<File[]>([]);
+	// const [additionalQuestion, setAdditionalQuestion] = useState("");
+	const [photosVideos, setPhotosVideos] = useState<File[]>([]);
```

Comment:
remove these comments if they are not being used

Reply:
removed